src/ralph/api/routers/statements.py

```diff
@@ -138,11 +138,13 @@ async def get(
     ###
     # Query string parameters defined by the LRS specification
     ###
-    # pylint: disable=invalid-name
-    statementId: Optional[str] = Query(None, description="Id of Statement to fetch"),
-    # pylint: disable=invalid-name, unused-argument
```

Comment:
this param wasn't unused actually 🤗 did I miss smth?

Reply:
Probably an oversight!

Reply:
Yes it is! You have several occurences of `statementId` variable in this method. What made you notice this ?

Reply:
It's an oversight from the first implementation of the API, indeed. Then @Leobouloc added a support to return a 400 bad request if voidedStatementId is given as a parameter